scripts/a11y-sla.sh

```diff
@@ -0,0 +1,146 @@
+          RESPONSE=$(curl -s -H "Authorization: Bearer $GITHUB_TOKEN" \
+            -H "Accept: application/vnd.github.v3+json" \
+            "https://api.github.com/repos/$GITHUB_REPOSITORY/issues?state=open&labels=$LABELS_QUERY&per_page=100&page=$PAGE")
+
+          # Break if no more issues
+          if [[ "$(echo "$RESPONSE" | jq '. | length')" -eq 0 ]]; then
+            echo "✅ No more issues to fetch. Exiting loop."
+            break
+          fi
+
+          # Merge response
+          ALL_ISSUES=$(echo "$ALL_ISSUES $RESPONSE" | jq -s 'add')
+
+          ((PAGE++))
+        done
+
+        # Get issue count
+        TOTAL_ISSUES=$(echo "$ALL_ISSUES" | jq '. | length')
+        echo "📊 Total Issues Fetched: $TOTAL_ISSUES"
+
+        if [[ "$TOTAL_ISSUES" -eq 0 ]]; then
+          echo "⚠️ No issues found with the required labels."
+          exit 0
+        fi
+
+        declare -A GROUPED_ISSUES
+
+        # Process issues
+        mapfile -t ISSUES < <(echo "$ALL_ISSUES" | jq -c '.[]')
+
+        for issue in "${ISSUES[@]}"; do
+          CREATED_AT=$(echo "$issue" | jq -r '.created_at')
+          ISSUE_NUMBER=$(echo "$issue" | jq -r '.number')
+
+          # Convert date to readable format
+          CREATED_DATE=$(date -d "$CREATED_AT" +"%Y-%m-%d" 2>/dev/null || date -u -jf "%Y-%m-%dT%H:%M:%SZ" "$CREATED_AT" +"%Y-%m-%d")
+
+          # Calculate issue age in weeks
+          CREATED_TIMESTAMP=$(date -d "$CREATED_AT" +%s 2>/dev/null || date -u -jf "%Y-%m-%dT%H:%M:%SZ" "$CREATED_AT" +%s)
+          DAYS_OLD=$(( (CURRENT_TIMESTAMP - CREATED_TIMESTAMP) / 86400 ))
+          WEEKS_OLD=$(( DAYS_OLD / 7 ))
+
+          # Fetch issue labels
+          LABELS=$(echo "$issue" | jq -r '[.labels[].name] | join(", ")')
+
+          # Determine impact level
+          IMPACT_LEVEL=""
+          for level in "${!LABEL_THRESHOLDS[@]}"; do
+            if [[ "$LABELS" == *"$level"* ]]; then
+              IMPACT_LEVEL="$level"
+              break
+            fi
+          done
+
+          if [[ -z "$IMPACT_LEVEL" ]]; then
+            echo "⚠️ Issue #$ISSUE_NUMBER has no recognized impact level (Blocker, Critical, Serious, Moderate). Skipping."
+            continue
+          fi
+
+          # Define SLA thresholds based on impact
+          IMPACT_SLA=${LABEL_THRESHOLDS[$IMPACT_LEVEL]}
+
+          # Determine the correct SLA label
+          NEW_SLA=""
+          if (( WEEKS_OLD >= IMPACT_SLA )); then
+            NEW_SLA="SLA Breach"
+          elif (( WEEKS_OLD >= IMPACT_SLA - 1 )); then
+            NEW_SLA="SLA P1"
+          elif (( WEEKS_OLD >= IMPACT_SLA - 2 )); then
+            NEW_SLA="SLA P2"
+          elif (( WEEKS_OLD >= IMPACT_SLA - 3 )); then
+            NEW_SLA="SLA P3"
+          fi
+
+          # Check current SLA labels and store them in an array for removal
+          CURRENT_SLA=""
+          REMOVE_LABELS=()
+
+          for sla_label in "${SLA_LABELS[@]}"; do
+            if [[ "$LABELS" == *"$sla_label"* ]]; then
```

Comment:
Same as above. Consider doing this:

```suggestion
            if [[ "${LABELS,,}" == "${sla_label,,}" ]]; then
```